index.d.ts

```diff
@@ -14,6 +14,7 @@ export interface Item {
     value: any;
     key?: string | number;
     color?: string;
+    testID?: string;
```

Comment:
Thanks for opening the PR @vinibgoulart! May I ask what problem this solves?

Reply:
Sorry, I didnt open a issue for this, actually it's a additional for this PR: #498.
When you try to pass the testID prop for the item, the typescript doesnt understand that there is this prop in Item.

Reply:
Gotcha makes sense!